stbt-run

```diff
@@ -22,7 +25,7 @@ parser.add_argument(
     '--save-video', help='Record video to the specified file', metavar='FILE',
     default=stbt.get_config('run', 'save_video'))
 parser.add_argument(
-    '--save-trace', metavar='FILE', default=None,
+    '--save-trace', metavar='FILE', nargs='?', const=TO_SOCKET,
```

Comment:
> we or someone else were to push our own tracing functionality upstream

I can't make technical judgements based upon the contents of patches that I've never seen.